src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2107,6 +2050,8 @@ theorem mul_omega_dvd {a : ordinal}
   (a0 : 0 < a) (ha : a < omega) : ∀ {b}, omega ∣ b → a * b = b
 | _ ⟨b, rfl⟩ := by rw [← mul_assoc, mul_omega a0 ha]
 
+/- This will be part of a future PR on multiplicative principal ordinals.
```

Comment:
Am I right that this is being removed without a replacement for now? Is the future PR ready or nearly ready? I don't mind introducing the regression if the replacement is coming soon. But if this code is broken then it would be better to just delete it.

Reply:
Yep, no replacement for now. The future PR is ready: #11701